runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -82,6 +82,9 @@
 	private static final String REMOVE_STATE = "Remove from canvas";
 	private static final String ADD_STATE = "Add to canvas";
 
+	private final EmptyBorder DEFAULT_PROGRESS_WRAPPER_BORDER = new EmptyBorder(0, 7, 7, 7);
+	private final int PROGRESS_SKILL_ICON_POSITION = 0;
```

Comment:
(committing via GitHub's UI)
```suggestion
	private static final int PROGRESS_SKILL_ICON_POSITION = 0;
```